src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -33,4 +62,31 @@ public void connectWhenInvalidUserNameReturnsFalse() {
     public void connectWhenInvalidUserPasswordReturnsFalse() {
         assertFalse(databaseManager.connect("sqlcmd", "sqlcmd", "Notsqlcmd"));
     }
-}
+
+    @Test
+    public void getTableNamesWhenNoTablesInDbReturnsEmptyStringArray() throws SQLException {
+        String[] expected = new String[]{};
+        assertArrayEquals(expected, databaseManager.getTableNames());
+    }
+
+    @Test
+    public void getTableNamesWhenTwoTablesInDbReturnsStringArrayWithTableNames() throws SQLException {
```

Comment:
TableNamesArray may be? I don.t think it's properly to specify class of return value at all.
Anybody can see test variable to compare or original method and his signature.
I think 'String' or smth is extra.

Reply:
Thanks a lot for the remark! Good naming of test methods is my pain for now ))) Will be fixed in next commit